app/lib/benchmark/benchmark_grid.dart

```diff
@@ -39,13 +45,22 @@ class BenchmarkGrid implements OnInit, OnDestroy {
   Timer _reloadTimer;
   bool _isShowArchived = false;
   bool _userIsAuthenticated = false;
+  List<String> _values = ['master'];
+  String _selectedValue = 'master';
```

Comment:
I think `Uri.base.queryParameters['branch'] ?? 'master'` would allow users to share the performance page of the branch they are viewing. `set selectedValue(...)` will have to be updated to set the query parameter.

Reply:
(This was a request on the build dashboard branch pages)

Reply:
Add it in issue: https://github.com/flutter/flutter/issues/53714